Update salesforce-source.md (#4263)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/salesforce-source.md

<a href="/files/c2c/salesforce/example.tf" target="_blank">Download example</a>

## Releases

### 3.X.X

> Configuration required for upgrading source from 2.X.X to 3.X.X version: 

Reviewer: ```suggestion
```

Author: It is required, because there will be 2 types of uses.
1. Old having c2c version 2.X.X
2. New

this steps are only for old users

Reviewer: We are covering this in line number 141.

:::info
Changes are required to be made in the Salesforce Vendor portal. No adjustments need to be made to the source configuration in C2C Sumologic portal.
:::

> * Steps to perform

Reviewer: ```suggestion
Follow the below steps to upgrade the Salesforce source from version 2.X.X to 3.X.X:
```

>      * Login to Salesforce

Reviewer: ```suggestion
     * Log in to Salesforce platform.
```

>      * Left side Menu, Navigate to **Build > Create > Apps**.

Reviewer: ```suggestion
     * In the left side menu, navigate to **Build > Create > Apps**.
```

>      * Under **Connected Apps**, Find your connected app, click **Edit**.

Reviewer: ```suggestion
     * Under **Connected Apps**, find your connected app and click **Edit**.
```

<a href="/files/c2c/salesforce/example.tf" target="_blank">Download example</a>

## Releases

### 3.X.X

:::info

> Perform the configuration changes in the Salesforce vendor platform and no adjustments are needed in Sumo Logic platform.

Reviewer 2: ```suggestion
Perform the configuration changes in the Salesforce vendor platform. No adjustments are needed in Sumo Logic platform.
```

:::

Follow the below steps to upgrade the Salesforce source from version 2.X.X to 3.X.X:

> 1. Log In to the Salesforce platform.

Reviewer 2: ```suggestion
1. Log in to the Salesforce platform.
```

> 1. Navigate to **Build** > **Create** > **Apps** in the left side menu.

Reviewer 2: ```suggestion
1.  From the left side menu, navigate to **Build** > **Create** > **Apps**.
```

> 1.  From the left side menu, navigate to **Build** > **Create** > **Apps**.

Reviewer 2: ```suggestion
1. From the left side menu, navigate to **Build** > **Create** > **Apps**.
```

1. Under **Connected Apps** section, find your connected app and click **Edit**.
1. Click the **Enable Client Credentials Flow** checkbox under **API (Enable OAuth Settings)**.
1. If prompted, accept the warning once you understand the security risks and click **Save**.
1. Navigate back to **Build** > **Create** > **Apps** in the left side menu.
1. Find your connected app and click **Manage**. ([Learn more](https://help.salesforce.com/s/articleView?id=sf.connected_app_client_credentials_setup.htm&type=5)).
1. Click **Edit Policies**.

> 1. Under **Client Credentials Flow**, go to **Run As** dropdown and click the search button. Find the user that you want to assign the client credentials flow.

Reviewer 2: ```suggestion
1. Under **Client Credentials Flow**, go to the **Run As** dropdown and click the search button. Find the user that you want to assign the client credentials flow.
```

    :::note

>     For Enterprise Edition orgs, we recommend you to select an execution user who has the API Only User permission.

Reviewer 2: ```suggestion
    For Enterprise Edition orgs, we recommend selecting an execution user who has the API Only User permission.
```
